docs/docsource/reference/introduction/supported_apis.rst

```diff
@@ -160,6 +160,30 @@ S3 API Endpoints
    | object PUT with v4 AUTH               |                |            |
    +---------------------------------------+----------------+------------+
 
+.. _IAM and Vault APIs:
+
+IAM and Vault APIs
+------------------
+
+The following S3 Identity Access Management (IAM) servies are available in
+|product|.
+
+.. tabularcolumns:: X{0.50\textwidth}X{0.30\textwidth}X{0.15\textwidth}
+.. table::
+   :widths: auto
+
+   +---------------------------------------+----------------+------------+
+   | Operation Name                        | Operation Type | Available? |
+   +=======================================+================+============+
+   | :ref:`create-access-key`              | Vault (POST)   | Yes        |
```

Comment:
Why is this vault and rest of them IAM?
Just curious.

Reply:
I may be confused on this point (and please correct me if I am}) I am listing "create-access-key" as Vault because I could not find this API in the IAM documentation, and assumed we were stepping out of line with IAM. 

To be clear: We have a module named Vault. It implements the IAM protocol. What would be best from the user's perspective to list these under? IAM, Vault, or some combination of the two?

Reply:
`IAM` for all calls which are listed here: https://docs.aws.amazon.com/IAM/latest/APIReference/API_Operations.html

And for all other calls, we use vaultclient ie, admin API.